src/modules/dataGrid.scss

```diff
@@ -26,18 +26,25 @@ $blackAlpha: rgba(0, 0, 0, .5);
         }
         &_table_component {
             .table-cell:last-child {
-                padding-right: inherit;
+                padding-right: 11px !important;
             }
             .table-header-cell:last-child {
-                padding-right: inherit;
-                .table-header-handle {
-                    cursor: pointer;
-                    float: right;
-                    position: relative;
-                    .ui.icon-size-small {
-                        width: 5px;
-                        svg {
+                padding-right: 11px !important;
+                position: relative;
+                .#{$prefixDrawerDataGrid}, .#{$prefixPageDataGrid}  {
+                    &_table_header_handle {
+                        cursor: pointer;
```

Comment:
Should we consider `cursor: col-resize;`?

Reply:
Yep, i like it.